shards/modules/core/wires.hpp

```diff
@@ -191,6 +191,11 @@ struct BaseRunner : public WireBase {
         }
       } else if (!wire->detached) {
         // but avoid stopping if detached and scheduled
+        if (wire->childWire) {
+          // stop the child wire if any
+          shards::stop(wire->childWire);
```

Comment:
Maybe better inside stop, because this is "flow" state kind of.
Check other stop usages if it makes sense to have there